app/android/app_template/src/org/xwalk/app/template/AppTemplateActivity.java

```diff
@@ -15,9 +15,23 @@
 import org.xwalk.app.XWalkRuntimeActivityBase;
 
 public class AppTemplateActivity extends XWalkRuntimeActivityBase {
+    private final int mUiOptions = View.SYSTEM_UI_FLAG_LAYOUT_STABLE |
+            View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION |
+            View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN |
+            View.SYSTEM_UI_FLAG_HIDE_NAVIGATION |
+            View.SYSTEM_UI_FLAG_FULLSCREEN |
+            View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY;
+
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        setSystemUiVisibilityChangeListener();
```

Comment:
Hi, we do not need to listen to system ui visibility when setIsFullscreen(true); not called.
So please put setSystemUiVisibilityChangeListener into setSystemUiVisibilityChangeListener:
    public void setIsFullscreen(boolean isFullscreen) {
        if (isFullscreen) {
            enterFullscreen();
            setSystemUiVisibilityChangeListener();
        }
    }